Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/DoubleTapMediaControlPlugin.swift

```diff
@@ -0,0 +1,49 @@
+import UIKit
+
+public class DoubleTapMediaControlPlugin: DoubleTapPlugin {
```

Comment:
Is it ok for a plugin that will be added to media control, to be a descendant from `UICorePlugin`?

Reply:
It seems to be an architectural limitation, @jademcosta. Besides having MediaControl on its name, this plugin is added to core instead.